reference/sql/attributes/auto-random.md

```diff
@@ -75,18 +75,24 @@ After creating the table, use the `SHOW WARNINGS` statement to see the maximum n
 show warnings
 ```
 
-```
-+-------+------+------------------------------------------------+
-| Level | Code | Message                                        |
-+-------+------+------------------------------------------------+
-| Note  | 1105 | Available implicit allocation times: 268435455 |
-+-------+------+------------------------------------------------+
+```sql
++-------+------+----------------------------------------------------------+
+| Level | Code | Message                                                  |
++-------+------+----------------------------------------------------------+
+| Note  | 1105 | Available implicit allocation times: 1152921504606846976 |
++-------+------+----------------------------------------------------------+
 ```
 
-In addition, for tables with the `AUTO_RANDOM` attribute, the value of the corresponding `TIDB_ROW_ID_SHARDING_INFO` column in the `information_schema.tables` system table is `PK_AUTO_RANDOM_BITS=x`. `x` is the number of shard bits.
+> **Note:**
```

Comment:
```suggestion
> **Warning:**
```

Reply:
`Warning` here might be a little too much. I have confirmed with the author @tangenta that `Note` is better. He will create a PR to change the wording in the Chinese version.